src/modules/database/entities/socialLinks.entity.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Column, Entity, ManyToOne, PrimaryGeneratedColumn,
+} from 'typeorm';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { IsOptional, IsString } from 'class-validator';
+import { Exclude } from 'class-transformer';
+import { User } from './user.entity';
+
+@Entity({ name: 'socialLink' })
+export class SocialLink {
+    @PrimaryGeneratedColumn('uuid')
+    @Exclude()
+    readonly id: string;
+
+    @Column({ type: 'varchar', length: 300 })
+    @ApiPropertyOptional({ description: 'social network name', example: 'Instagram' })
+    @IsOptional()
+    @IsString()
+    name: string;
+
+    @Column({ type: 'varchar', length: 300 })
+    @ApiPropertyOptional({
+      description: 'link to user account',
+      example: 'https://www.instagram.com/userProfile',
+    })
+    @IsOptional()
+    @IsString()
+    link: string;
+
+    @ManyToOne(() => User, (user) => user.answers, { onDelete: 'CASCADE' })
+    user: User;
```

Comment:
what is it?!?!

Reply:
This is the entity that represents the social networks of the user in the form of a record: 
 "socialLinks": [
    {
      "name": "Instagram",
      "link": "https://www.instagram.com/userProfile"
    }

Reply:
This is a relationship to the table that stores the user's social links. For exemple:
"socialLinks": [
      {
        "name": "Instagram",
        "link": "https://www.instagram.com/userProfile"
      }
    ]

Reply:
done